src/react-elastic-carousel/components/Pagination/Dot.js

```diff
@@ -37,6 +37,7 @@ class DotContainer extends React.Component {
     const { active } = this.props;
     return (
       <Dot
+        type="button"
```

Comment:
I would do it in the `Dot` component itself (line 10) in case we use `Dot` in other places:

```jsx
const Dot = styled.button.attrs(() => ({ type: "button" }))`
  transition: all 250ms ease;
  border: none;
  margin: 5px;
  // ...
`
```